packages/core/src/helpers/chain.ts

```diff
@@ -3,16 +3,27 @@ import { DEFAULT_SUPPORTED_CHAINS, ChainId } from '../constants'
 export const getChainById = (chainId: ChainId) =>
   DEFAULT_SUPPORTED_CHAINS.find((network) => network.chainId === chainId)
 
-export const getExplorerAddressLink = (address: string, chainId: ChainId): string | '' =>
-  getChainById(chainId)?.getExplorerAddressLink(address) || ''
+export const getExplorerAddressLink = (address: string, chainId: ChainId): string | '' => {
+  console.warn('getExplorerAddressLink is deprecated, can call with Chain directly')
+  return getChainById(chainId)?.getExplorerAddressLink(address) || ''
+}
 
-export const getExplorerTransactionLink = (transactionHash: string, chainId: ChainId): string | '' =>
-  getChainById(chainId)?.getExplorerTransactionLink(transactionHash) || ''
+export const getExplorerTransactionLink = (transactionHash: string, chainId: ChainId): string | '' => {
```

Comment:
As we deprecating all those methods, I think we should introduce new hook, like useNetwork() that will return current network

Reply:
since `getChainById` still works through DEFAULT_SUPPORTED_CHAINS, can add so in the next major version

Reply:
👍